lib/generators/doorkeeper/openid_connect/templates/initializer.rb

```diff
@@ -24,9 +26,12 @@
     # redirect_to new_user_session_url
   end
 
-  subject do |resource_owner|
+  subject do |resource_owner, application|
     # Example implementation:
-    # resource_owner.key
+    # resource_owner.id
+
+    # or if you need pairwise subject identifier, implement like below:
+    # Digest::SHA256.hexdigest("#{resource_owner.id}#{URI.parse(application.redirect_uri).host}#{'your_secret_salt'}")
```

Comment:
Hmm ok I see now that it's not as straightforward as I thought, since in `IdToken#subject` we have no way of knowing which algorithm we should use (if we had both a `subject` and a `pairwise_subject`).

My concern is breaking configuration in future releases, and also users setting `subject_types_supported [:pairwise]` but not actually implementing a pairwise subject algorithm, but both of these are not real dealbreakers. Let me know if you see another solution, otherwise I think we can merge this.

One last request: Can you squash the commits into one, with a `feature:` prefix? (sorry I recently fixed the docs which said `feat`, but I've always used `feature` ;-))